src/ApiCalls.ts

```diff
@@ -113,3 +114,34 @@ export async function removeFavoriteCamp(favoriteId: number) {
     throw error;
   }
 }
+
+export async function getCampgroundReviews(id: string) {
+  try {
+    const response = await fetch(`https://backcountry-bookings-be.herokuapp.com/api/v1/reviews?campsite_id=${id}`)
+    if (!response.ok) {
+      throw new Error(`Get campground reviews request failed with status ${response.status}`)
+    }
+    const campReviews = await response.json();
+    return campReviews;
+  } catch (error) {
+    console.log(`Failed to fetch campground reviews: ${error}`)
+    throw error;
+  }
+}
+
+export async function postCampgroundReview(reviewObj: BodyInit, campID: string) {
+  try {
+    const response = await fetch(`https://backcountry-bookings-be.herokuapp.com/api/v1/reviews?user_id=1&campsite_id=${campID}`, {
+      method: "POST",
+      body: reviewObj
+    })
+    if (!response.ok) {
+      return response.text().then(text => { throw new Error(text) })
+    }
+    const reviewResp = await response.json();
+    return reviewResp;
+  } catch (error) {
+    console.log(`Failed to post campground review: ${error}`)
```

Comment:
If you like console.logs that totally fine - but sometimes I've seen people use console.error for situations just like this. Feel free to try it for fun. 

I think the argument for it is that it helps separate what you want to _leave in_ for production and what you want to remove. Typically, you don't want any console logs running in production.